opentelemetry-api/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Remove grpc status codes ([#1282](https://github.com/open-telemetry/opentelemetry-python/pull/1282))
```

Comment:
Not a huge deal, but I would argue this change is better described as "renaming and updating status codes to adhere to spec" rather than grpc specifically.